src/routes/blog/posts/+layout.svelte

```diff
@@ -13,7 +13,7 @@
 </script>
 
 <Metadata
-	title={$_("metadata.blog_page", defaultI18nValues)}
+	title={$_("metadata.blog_page", { values: { ...defaultI18nValues, title }})}
```

Comment:
This does not seem to fix the issue does it?

Reply:
yeah, idk why, I'll try again